File: Etl/HealthController.py
from flask import Flask, jsonify
app = Flask(__name__)

@app.route('/health', methods=['GET'])
def health():
    return jsonify(status='ETL Healthy', time=str(__import__('datetime').datetime.utcnow()))

if __name__ == '__main__':
    app.run(port=5100)

File: Api/HealthController.cs
using Microsoft.AspNetCore.Mvc;

namespace Api.Controllers
{
    [ApiController]
    [Route("health")]
    public class HealthController : ControllerBase
    {
        [HttpGet]
        public IActionResult Get() => Ok(new { status = "Healthy", time = System.DateTime.UtcNow });
    }
}
